sidebarsDocs.js

```diff
@@ -23,528 +23,652 @@ const sidebars = {
+          },
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Consuming Messages',
+        link: {type: 'doc', id: 'admin-guide'},
+        items: [
+          {
+            type: 'doc',
+            id: 'consumers',
+            label: 'Describing Consumers',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-cancel',
+            label: 'Consumer Cancellation Notifications',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-prefetch',
+            label: 'Consumer Prefetch',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-priority',
+            label: 'Consumer Priorites',
+          },
+        ],
+      },
+      {
+        type: 'doc',
+        id: 'confirms',
+        label: 'Consumer Acknowledgements and Publisher Confirms',
+      },
+      {
+        type: 'category',
+        label: 'Working with High Availability Queues and Streams',
+        items: [
+          {
+            type: 'doc',
+            id: 'queues',
+            label: 'Queues in General',
+          },
+          {
+            type: 'doc',
+            id: 'quorum-queues',
+            label: 'Quorum Queues',
+          },
+          {
+            type: 'doc',
+            id: 'classic-queues',
+            label: 'Classic Queues',
+          },
+          {
+            type: 'doc',
+            id: 'classic-queues',
+            label: 'Migrate Mirrored Classic Queues to Quorum Queues',
+          },
+          {
+            type: 'doc',
+            id: 'ttl',
+            label: 'Queue and Message TTL and the Expiration Feature',
+          },
+          {
+            type: 'doc',
+            id: 'maxlength',
+            label: 'Set the Queue Length',
+          },
+          {
+            type: 'doc',
+            id: 'lazy-queues',
+            label: 'Set the Queue Modes: Lazy and Default',
+          },
+          {
+            type: 'doc',
+            id: 'dlx',
+            label: 'Dead Lettering Feature',
+          },
+          {
+            type: 'doc',
+            id: 'priority',
+            label: 'Turn your Queues into Priority Queues',
+          },
+          {
+            type: 'doc',
+            id: 'streams',
+            label: 'Streams',
+          },
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Clustering',
+        link: {type: 'doc', id: 'clustering'},
+        items: [
+          {
+            type: 'doc',
+            id: 'cluster-formation',
+            label: 'Cluster Formation and Peer Discovery',
+          },
+        ]
+      },
+      {
+        type: 'category',
+        label: 'Working with Plugins',
+        items: [
+          {
+            type: 'doc',
+            id: 'installing-plugins',
+            label: 'Installing Plugins',
+          },
+          {
+            type: 'doc',
+            id: 'plugin-development',
+            label: 'Plugin Development',
```

Comment:
This is not an admin topic.

I would move that to "other-information" until we know what to do with it, perhaps. Or unlink it from the ToC.

Reply:
Done